src/query/functions/src/scalars/array.rs

```diff
@@ -744,4 +754,38 @@ fn register_array_aggr(registry: &mut FunctionRegistry) {
             }))
         });
     }
+
+    for (fn_name, sort_desc) in ARRAY_SORT_FUNCTIONS {
+        registry.register_passthrough_nullable_1_arg::<EmptyArrayType, EmptyArrayType, _, _>(
+            fn_name,
+            FunctionProperty::default(),
+            |_| FunctionDomain::Full,
+            vectorize_1_arg::<EmptyArrayType, EmptyArrayType>(|arr, _| arr),
+        );
+
+        registry.register_passthrough_nullable_1_arg::<ArrayType<GenericType<0>>, ArrayType<GenericType<0>>, _, _>(
+            fn_name,
+            FunctionProperty::default(),
+            |_| FunctionDomain::Full,
+            vectorize_1_arg::<ArrayType<GenericType<0>>, ArrayType<GenericType<0>>>(|arr, _| {
+                let sort_desc = vec![SortColumnDescription {
+                    offset: 0,
+                    asc: sort_desc.0,
+                    nulls_first: sort_desc.1,
+                }];
+                let columns = vec![arr];
+                let len = columns.get(0).map_or(1, |c| c.len());
```

Comment:
Can use `let len = arr.len();`.
Also check if length is zero or one to rturn directly.